src/components/WwwFrame/TheHeader.vue

```diff
@@ -653,12 +653,14 @@ export default {
 				logReadQueryError(e, 'User Data For Optimizely Metrics');
 			}
 		}
+	},
+	mounted() {
```

Comment:
Just out of curiosity, why was this move needed?

Reply:
We're having to take some stabs in the dark to try to resolve issues with getting the hotjar surveys to display. With this we're testing if ensuring the attributes are only set in the browser and that they're called on every page load will make a difference.